src/mesh/parallel/identity.cxx

```diff
@@ -0,0 +1,11 @@
+#include "bout/paralleltransform.hxx"
+#include "bout/mesh.hxx"
+
+void ParallelTransformIdentity::calcYUpDown(Field3D& f) {
```

Comment:
I think the other transforms provide a Field2D form as well which does nothing (although I might be confused with to/from field aligned).

Reply:
It looks like that is proposed in #1432

Reply:
It looks like I was thinking of the field aligned shifts in `Mesh` https://github.com/boutproject/BOUT-dev/blob/next/include/bout/mesh.hxx#L686